docs/pipeline.md

```diff
@@ -70,33 +70,131 @@ dependencies between components to build the DAG.
 Currently Fondant supports linear DAGs with single dependencies. Support for non-linear DAGs will be
 available in future releases.
 
-## Compiling and Running a pipeline
+## Compiling a pipeline
 
 Once all your components are added to your pipeline you can use different compilers to run your
 pipeline:
 
 !!! note "IMPORTANT"
 When using other runners you will need to make sure that your new environment has access to:
 
-- The base path of your pipeline (as mentioned above)
-- The images used in your pipeline (make sure you have access to the registries where the images are
-  stored)
+    - The base path of your pipeline (as mentioned above)
+    - The images used in your pipeline (make sure you have access to the registries where the images are
+    stored)
+
+=== "Console"
+    
+    === "Local Compiler"
```

Comment:
If we rename this to "Local", "Vertex", and "Kubeflow" for both the compilers and runners, the tabs will be linked.

Reply:
Good point, changed it